test/extensions/filters/http/cache/cache_filter_test.cc

```diff
@@ -114,19 +131,85 @@ class CacheFilterTest : public ::testing::Test {
     ::testing::Mock::VerifyAndClearExpectations(&decoder_callbacks_);
   }
 
+  // This function assumes that there is a body created using std::string(body_size, 'a').
+  void testSuccessfulValidation(CacheFilterSharedPtr filter, uint64_t body_size) {
+    ASSERT(body_size > 0);
+
+    // Make request require validation
+    request_headers_.setReferenceKey(Http::CustomHeaders::get().CacheControl, "no-cache");
```

Comment:
see `CustomHeaderValues::CacheControlValues::NoCache` in source/common/http/headers.h

Reply:
Done.